terraform/fleetspeak_configurator/build_configs.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/google/fleetspeak/fleetspeak/src/e2etesting/lib"
+	"os"
+)
+
+var (
+	configDir     = flag.String("config_dir", "", "Directory to put config files")
+	numClients    = flag.Int("num_clients", 1, "Number of clients")
+	numServers    = flag.Int("num_servers", 1, "Number of servers")
+	mysqlAddress  = flag.String("mysql_address", "", "MySQL server address")
+	mysqlDatabase = flag.String("mysql_database", "", "MySQL database name to use")
+	mysqlUsername = flag.String("mysql_username", "", "MySQL username to use")
+	mysqlPassword = flag.String("mysql_password", "", "MySQL password to use")
+)
+
+func run() error {
+	serverHosts := make([]string, *numServers)
+	for i := 0; i < *numServers; i++ {
+		fmt.Scanf("%s", &serverHosts[i])
+	}
+
+	err := setup.BuildConfigurations(*configDir, serverHosts, *numClients,
```

Comment:
Where is setup declared? I don't understand how this can compile.

Reply:
It was the package name imported with "github.com/google/fleetspeak/fleetspeak/src/e2etesting/lib". The name and the last element of the path (lib) were different, but it's already fixed in the new commit.